src/apps/relay/netengine.c

```diff
@@ -943,73 +950,29 @@ static ioa_engine_handle create_new_listener_engine(void) {
 static void *run_udp_listener_thread(void *arg) {
   static int always_true = 1;
```

Comment:
erm.... this is not a great name for a variable...?
especially since it seems this specific variable does not accomplish anything.